src/Context.php

```diff
@@ -210,4 +212,13 @@ public function offsetUnset(mixed $offset): void
     {
         throw new \LogicException('Context is immutable');
     }
+
+    public static function fromPath(string $path): self
+    {
+        $dotenv = new Dotenv();
+        $dotenv->loadEnv($path);
+        unset($_ENV['SYMFONY_DOTENV_VARS']);
```

Comment:
What about returning the $env, instead of passing by $_ENV?
(ie, use the [`parse()`](https://github.com/symfony/dotenv/blob/ceadb434fe2a6763a03d2d110441745834f3dd1e/Dotenv.php#L227) method

Reply:
I would prefer to keep it that way so we can overload the dotenv file with a `.local` file as it should be.

Reply:
You are right 👍🏼 I was a bit worried to pollute $_ENV with things not really related... but if we cannot, let's do it